pkg/api/api/projects.go

```diff
@@ -0,0 +1,54 @@
+package api
```

Comment:
The double `api/api` directory is weird to me.

Couldn't we do:

```
pkg/api/
  projects.go
  gen/...
```

I'm also tempted to call it `apiclient` instead of just `api`? 🤷

Reply:
Sure. I originally called it `client` but I really dislike the name.

I think putting it in top level sounds good.